akka-actor-typed-tests/src/test/java/jdocs/akka/typed/IntroTest.java

```diff
@@ -104,6 +106,37 @@ public Start(String name) {
   }
   //#hello-world-main
 
+  public abstract static class CustomDispatchersExample {
+    private CustomDispatchersExample() {
+    }
+
+    public static class Start {
+      public final String name;
+
+      public Start(String name) {
+        this.name = name;
+      }
+    }
+
+    //#hello-world-main-with-dispatchers
+    public static final Behavior<Start> main =
+      Behaviors.setup( context -> {
```

Comment:
weird formatting space, can we remove it quickly in case people would complain about weird formatting?